scripts/synthtrace.js

```diff
@@ -6,5 +6,4 @@
  * Side Public License, v 1.
  */
 
-require('../src/setup_node_env/node_version_validator');
-require('@elastic/apm-synthtrace/bin/synthtrace');
```

Comment:
Running a script like `node scripts/synthtrace simple_trace --local` doesn't work for me. It seems that the path for the scenarios resolves in something like /private/var/tmp/_bazel_giorgosb/.../execroot/kibana/bazel-out/darwin-fastbuild/bin/packages/elastic-apm-synthtrace/target_node/scenarios/simple_trace.ts

Reply:
Thanks for the heads up. 
I can't make this work properly if I import the package as `require('@elastic/apm-synthtrace')`. I've reverted to importing from the package directly